### ISSUE TITLE
Como analisar as indicações

### ISSUE THREAD

**Issue by woliveiras** [training-center/advocacy#2]
O Core Team será responsável por analisar as indicações das pessoas.

Devemos especificar nesta issue quais os pontos que o Core deverá analisar.

Devemos discutir aqui todos os pontos referentes a essa especificação e somente estes pontos, caso tenhamos algo a mais para discutir devemos abrir outra issue.

**Comment by WendellAdriel** [comment]
A análise será um processo manual, pois não teremos como automatizar uma grande parte do processo. Então creio que os pontos que a `Core Team` deve analisar são:

- Verificar se o candidato cumpre os requisitos base para o título indicado, como descrito na https://github.com/training-center/advocacy/issues/3

- Verificar se o candidato é ativo na comunidade, ou seja, contribui/participa do TC pelo menos duas vezes na semana com atividades relacionadas ao título que foi indicado

- Avaliar a forma como a pessoa interage com outras pessoas na comunidade e fora da comunidade (nas redes sociais por exemplo), ou seja, se ela trata as pessoas com respeito, se a pessoa não discrimina nem posta coisas preconceituosas nas redes sociais e verificar se não infringe nenhum ponto do **[código de conduta](http://bit.ly/tc-codigo-de-conduta)**

- Se depois de analisar todos os pontos acima ainda existir alguma dúvida sobre a aprovação do candidato, o `Core Team` deve tentar marcar uma pequena "reunião" com o mesmo para esclarecer demais pontos em dúvida.

**Comment by Diego-Brocanelli** [comment]
Um ponto que acredito ser interessante é a votação em si.

Como será realizado o processo de votação?

Um método que creio ser bacana é ***50% + 1*** do core team.

**Comment by Diego-Brocanelli** [comment]
O processo de análise individual/coletivo será documentado pelo core team?

**Comment by WendellAdriel** [comment]
Sim, a aprovação deve ser por maioria, ou seja, com a regra que disse: **50% + 1**

Sobre o processo de votação, para não gerar nenhum problema creio que pode ser criado um **Form anônimo** em que a `Core Team` irá votar com apenas duas questões:

- Concorda com a aprovação?
- Explique resumidamente o motivo

E no final da "contagem" de votos, o resultado ser apresentado na issue.

**Comment by Diego-Brocanelli** [comment]
@WendellAdriel  bacana o processo, pois o form será um documento do TC.

Nesse form podemos inserir,
- Nome do candidato.
- Sim ou Não.
- Motivo do voto escolhido.

Pois assim será possível filtrar pelo candidato e o resultado da votação.

E caso não seja aprovado o candidato indicado, será possível dar feedback para a comunidade do resultado, já com os pontos observados pelo TC.

**Comment by WendellAdriel** [comment]
Sim @Diego-Brocanelli creio que seja interessante fazer isso mesmo. Devemos apenas manter os votos como anônimos para evitar qualquer tipo de problemas.
